README.md

```diff
@@ -39,7 +40,7 @@ All of the services are running privately in the aws elastic kubernetes cluster
 
  2. Configuration
  
-    Cluster options could be configured by modifying `cluster-config/eks-cluster-config.json` file. Please ensure that the region set in the `cluster-config/eks-cluster-config.json` must match the region set for aws cli (~/.aws/config).
+    Cluster options could be configured by modifying `infrastructure-config/infra-config.json` file. Please ensure that the region set in the `infrastructure-config/infra-config.json` must match the region set for aws cli (~/.aws/config).
```

Comment:
Grammar: either remove "must" or replace "ensure" with "note": 
a) Please ensure that ... matches ...
b) Please note that ... must match ...

Reply:
Fixed